deno.jsonc

```diff
@@ -35,7 +35,10 @@
     "@std/yaml": "jsr:@std/yaml@^1.0.5",
     "is-what": "https://deno.land/x/is_what@v4.1.15/src/index.ts",
     "outdent": "jsr:@cspotcode/outdent@^0.8",
-    "pkgx": "https://deno.land/x/libpkgx@v0.20.1/mod.ts",
-    "pkgx/": "https://deno.land/x/libpkgx@v0.20.1/src/"
+    "pkgx": "https://deno.land/x/libpkgx@v0.20.3/mod.ts",
+    "pkgx/": "https://deno.land/x/libpkgx@v0.20.3/src/",
+    // test dependencies
+    "faker": "npm:@faker-js/faker@^9.4.0",
+    "npm:@types/node@*": "npm:@types/node@*" // implicit
```

Comment:
Without this line, `deno install` would not add this dependency to the lockfile. However, after running tests, the dependency would end up being added there.

Now, `deno install` (which is like `npm install` after deno v2) will properly update/check the lockfile.